mavisp/modules.py

```diff
@@ -111,21 +110,20 @@ def ingest(self, mutations):
         method = tmp[0]
 
         tmp = os.listdir(os.path.join(self.data_dir, self.module_dir, f'{structure_ID}_{residue_range}', method))
-        if len(tmp) != 1:
-            raise MAVISpMultipleError(warning=warnings,
-                                      critical=[MAVISpCriticalError(this_error)])
-        model = tmp[0]
+        results = []
+        for technique in tmp:
+            model = technique
+            method_dirs = os.listdir(os.path.join(self.data_dir, self.module_dir, f'{structure_ID}_{residue_range}', method, model))
```

Comment:
I think you should append the methods as a list here - we want this to capture *all* the methods.
Currently you're just considering the last one since `method_dirs` is overwritten at every time cycle of the for loop